docs/tsconfig.json

```
@@ -3,6 +3,7 @@
   "extends": "@tsconfig/docusaurus/tsconfig.json",
```

```suggestion
  "extends": "@docusaurus/tsconfig",
```

You will need to install this: `yarn add --dev `@docusaurus/tsconfig`

---

See https://github.com/facebook/docusaurus/pull/9050

---

Hmmm, the lock file looks quite different now. Could that be a problem?

---

That is expected, should be an issue